### PR TITLE
Flash sale

### PR DESCRIPTION
**Pull request by jkerr321**
@lc512k @tavvy 

Update to marketing promo to offer 25% discount and update to Lionel slider to show 33% discount. Merging this behind a flag so QA can look at it while I finish it off.

Known issue: price in Lionel slider is currently not right

promo slider

### CODE REVIEW

File: subscription-offer-prompt/lionel.js [hunk 1]
```diff
@@ -108,16 +109,16 @@ const getPrice = countryCode => {
 	return utils.toCurrency.apply(null, prices[countryCode] || prices.default);
 };
 
-const getSubscriptionPromptValues = countryCode => {
+const getSubscriptionPromptValues = (countryCode, flags) => {
 	const price = getPrice(countryCode);
-	if (countryCode === 'USA') {
-		return { discount: 33, offerId: 'a9582121-87c2-09a7-0cc0-4caf594985d5', price };
+	if (countryCode === 'USA' || flags.get('priceFlashSale')) {
```
Comment (tavvy) [inline subscription-offer-prompt/lionel.js]: So the offer for is all of the USA **or** if the flag is on?

Comment (jkerr321) [inline subscription-offer-prompt/lionel.js]: Yep :)

**Comment by jkerr321** [comment]
@tavvy I've hardcoded the 33% discount so that we can get this out and added some more tests. I'm going to work on fetching the price from the API in a separate branch. Pls let me know if you've got any comments on the latest changes then I'll merge :)

File: subscription-offer-prompt/lionel.js [hunk 2]
```diff
@@ -93,24 +93,39 @@ const createSubscriptionPrompt = values => {
...
+			CHE: [439, 'CHF'],
+			GBR: [355, 'GBP'],
+			HKG: [3295, 'HKD'],
+			JPN: [583, 'JPN'],
+			SGP: [555, 'SGD'],
+			USA: [429, 'USD'],
+			default: [395, 'EUR']
+		};
+	} else {
+		console.log("hanging out no flag");
```
Comment (tavvy) [inline subscription-offer-prompt/lionel.js]: Does this need to be removed?

File: subscription-offer-prompt/test/lionel.spec.js [hunk 3]
```diff
@@ -87,12 +105,112 @@ describe('"Lionel Slider" Subscription Offer Prompt', () => {
...
 
 	it('should not show barrier page has not been visited in this session', () => {
 		sessionStorage.unset('last-seen');
-		return init().then(popup => should.not.exist(popup));
+		return init(flags).then(popup => should.not.exist(popup));
+	});
+
+});
+
+describe('"Lionel Slider" Subscription Offer Prompt - USA', () => {
```
Comment (tavvy) [inline subscription-offer-prompt/test/lionel.spec.js]: The way describe block nesting works means that you could define the stubs and fixtures that won't change at the top of the file in the top level describe block.

That way you only have to define the stubs you want to change (country) in the nested blocks. And not have to redefine additional stubs each time.

Comment (jkerr321) [inline subscription-offer-prompt/test/lionel.spec.js]: oooo that's exciting!

**Pull request merged**